conbench/entities/benchmark_result.py

```diff
@@ -358,57 +358,53 @@ class BenchmarkResultCreate(marshmallow.Schema):
         },
     )
     median = marshmallow.fields.Decimal(
         required=False,
         metadata={
-            "description": "The median from `data`, will be calculdated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
+            "description": "The median from `data`, will be calculated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
         },
     )
     stdev = marshmallow.fields.Decimal(
         required=False,
         metadata={
-            "description": "The standard deviation from `data`, will be calculdated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
+            "description": "The standard deviation from `data`, will be calculated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
         },
     )
     q1 = marshmallow.fields.Decimal(
         required=False,
         metadata={
-            "description": "The first quartile from `data`, will be calculdated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
+            "description": "The first quartile from `data`, will be calculated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
         },
     )
     q3 = marshmallow.fields.Decimal(
         required=False,
         metadata={
-            "description": "The third quartile from `data`, will be calculdated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
+            "description": "The third quartile from `data`, will be calculated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
         },
     )
     iqr = marshmallow.fields.Decimal(
         required=False,
         metadata={
-            "description": "The inter-quartile range from `data`, will be calculdated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
+            "description": "The inter-quartile range from `data`, will be calculated on the server if not present (the preferred method), but can be overridden if sent. Will be marked `null` if any iterations are missing."
         },
     )
 
 
-class BenchmarkResultSchema:
-    create = BenchmarkResultCreate()
```

Comment:
What is/was the value of this wrapping technique?

Reply:
I suspect it was reserving a spot for a future `update` schema. Not very important